NPE crash when config.xml "version" attr is broken

AppV 1.2.2.1 - From Gplay report:

Code: https://github.com/Catfriend1/syncthing-android/blob/b21ebccbd99ce3f695982e49ccf9d50c21f9311b/app/src/main/java/com/nutomic/syncthingandroid/util/ConfigXml.java#L344

```
Caused by: java.lang.NullPointerException: 
  at com.nutomic.syncthingandroid.util.ConfigXml.migrateSyncthingOptions (ConfigXml.java:344)
  at com.nutomic.syncthingandroid.util.ConfigXml.updateIfNeeded (ConfigXml.java:238)
  at com.nutomic.syncthingandroid.util.ConfigXml.loadConfig (ConfigXml.java:108)
  at com.nutomic.syncthingandroid.activities.FirstStartActivity.checkForParseableConfig (FirstStartActivity.java:593)
  at com.nutomic.syncthingandroid.activities.FirstStartActivity.onCreate (FirstStartActivity.java:107)
  at android.app.Activity.performCreate (Activity.java:7145)
  at android.app.Activity.performCreate (Activity.java:7136)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1272)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2933)
```

--- a/app/src/main/java/com/nutomic/syncthingandroid/util/ConfigXml.java
+++ b/app/src/main/java/com/nutomic/syncthingandroid/util/ConfigXml.java
@@ -337,7 +337,7 @@ public class ConfigXml {
         Folder defaultFolder = new Folder();
 
         /* Read existing config version */
-        int iConfigVersion = Integer.parseInt(mConfig.getDocumentElement().getAttribute("version"));
+        int iConfigVersion = getAttributeOrDefault(mConfig.getDocumentElement(), "version", 0);
         int iOldConfigVersion = iConfigVersion;
         LogV("Found existing config version " + Integer.toString(iConfigVersion));
 

